docs/book/src/cronjob-tutorial/testdata/emptycontroller.go

```diff
@@ -41,6 +42,7 @@ objects, so these are added out of the box.
 type CronJobReconciler struct {
 	client.Client
 	Log logr.Logger
+	Scheme *runtime.Scheme
```

Comment:
This file is used at the start of the book as an **empty** controller. The scheme is introduced later in [section 1.7](https://book.kubebuilder.io/cronjob-tutorial/controller-implementation.html). The corresponding file is `docs/book/src/cronjob-tutorial/testdata/project/controllers/cronjob_controller.go`. In that page it is explained why do we add the `Scheme`, so I again think that you should not add the `Scheme` in the **empty** controller.

Reply:
The empty controller now, after this section is written, will have the scheme attribute by . default so IMO makes sense the doc also be updated.

Reply:
I missed #1054. This change does make perfect sense.